skyportal/tests/frontend/test_user_management.py

```diff
@@ -62,6 +63,8 @@ def test_grant_and_delete_user_acl(driver, super_admin_user, user):
     filter_for_user(driver, user.username)
     driver.click_xpath(f'//*[@data-testid="addUserACLsButton{user.id}"]')
     driver.click_xpath('//*[@data-testid="addUserACLsSelect"]')
+    element = driver.find_element(By.XPATH, f'//li[text()="{acl}"]')
```

Comment:
Not sure why you need the by.xpath, that's already exactly what our custom driver offers. The `wait_for_xpath()` should return the element. Please remove that imports and use the existing syntax if possible